bokehjs/src/lib/core/kinds.ts

```diff
@@ -463,6 +545,10 @@ export namespace Kinds {
     override toString(): string {
       return `NonNegative(${this.base_type.toString()})`
     }
+
+    may_have_refs(): boolean {
+      return this.base_type.may_have_refs()
```

Comment:
What is an example of a case where this is true?

Reply:
Most likely it will never be true, but for the sake of completeness it's there. In the future I need to add some more structure to those classes, so this will be implemented the same for for all single parameterized kinds. I suppose it could be an optimization to just return `false`.